src/main/java/com/google/alpollo/SongHead.java

```diff
@@ -0,0 +1,23 @@
+package com.google.alpollo;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Index;
+import com.googlecode.objectify.annotation.Id;
+
+@Entity
+public class SongHead extends Song {
+  @Id
+  private String id;
+
+  public SongHead() { }
```

Comment:
you can make it `private` (or even `protected`?) since developers are not supposed to use it, it's just for Objectify